go.mod

```diff
@@ -1,15 +1,23 @@
 module github.com/aws/aws-xray-daemon
 
-go 1.16
+go 1.18
 
 require (
-	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
 	github.com/aws/aws-sdk-go v1.29.29
 	github.com/cihub/seelog v0.0.0-20170130134532-f561c5e57575
-	github.com/go-ole/go-ole v1.2.4 // indirect
 	github.com/shirou/gopsutil v2.19.10+incompatible
 	github.com/stretchr/testify v1.4.0
 	golang.org/x/net v0.0.0-20210716203947-853a461950ff
 	golang.org/x/sys v0.0.0-20210423082822-04245dca01da
```

Comment:
Can we bump the net and sys packages to latest as well?

Reply:
Done